building_map_tools/building_crowdsim/navmesh/object_manager.py

```diff
@@ -0,0 +1,9 @@
+class Manager:
+
+    def __init__(self):
+        self.data = []
+
+    def add_obj(self, obj):
+        assert(hasattr(obj, 'id')), "Adding object without id attribute!"
+        obj.id = len(self.data)
+        self.data.append(obj)
```

Comment:
I'm not too sure what's the idea behind this object, is it necessary?

Reply:
This is the base class for all the VerticesManager, LaneManager, PolygonManager... The idea is all these manager will initialize the id to each added instance.